File: recursion.c
#include <stdio.h>

void eatBanana(int bananas) {
    if (bananas < 1) return;
    printf("You ate a banana!\n");
    eatBanana(bananas - 1);
}

int main() {
    eatBanana(10);
    return 0;
}

File: recursion.cs
using System;

class Program
{
    static void Main()
    {
        EatBanana(10);
    }

    static void EatBanana(int bananas)
    {
        if (bananas < 1) return;
        Console.WriteLine("You ate a banana!");
        EatBanana(bananas - 1);
    }
}
